src/Atmos/Model/bc_energy.jl

```diff
@@ -35,8 +35,9 @@ function atmos_energy_boundary_state!(
 )
     @unpack aux⁻, state⁻, t = args
     FT = eltype(aux⁻)
-    _T_0::FT = T_0(atmos.param_set)
-    _cv_d::FT = cv_d(atmos.param_set)
+    param_set
```

Comment:
I see this line several times. Is it supposed to define the param_set from the getter, but the RHS is missing?

Reply:
Thanks, fixed.